docs/tf_model_binary_input.md

```diff
@@ -24,7 +24,7 @@ rmdir resnet_v2_fp32_savedmodel_NHWC/
 
 Convert the TensorFlow model to Intermediate Representation format using model_optimizer tool:
 ```Bash
```

Comment:
Since you change to lowercase ("Bash" -> "bash") in other files maybe you could also make such change here?

Reply:
Done:)